docker-images/kafka-based/kafka/scripts/tls_utils.sh

```diff
@@ -55,4 +55,30 @@ function find_ca {
             echo "$ca"
         fi
     done
+}
+
+# Parameters:
+# $1: Path to the new truststore
+# $2: Truststore password
+# $3: Base path where the certificates are mounted
+# $4: Environment variable defining the certs that should be loaded
+function prepare_truststore {
+    STORE=$1
+    PASSWORD=$2
+    BASEPATH=$3
+    TRUSTED_CERTS=$4
```

Comment:
Sorry one thing which came while reviewing the corresponding bridge PR, just about naming. If we know this function is about "truststore", it's clear `STORE` as name, but then why `TRUSTED_CERTS` instead of just `CERTS`. I have no strong opinion but I would go with `STORE` and `CERTS` or `TRUSTED_STORE` and `TRUSTED_CERTS`. Wdyt?

Reply:
(I can rename the `STORE` to `TRUSTSTORE` if you want, I used `STORE` as we seem to use it everywhere else, so I just copied it).

Reply:
I renamed the `STORE` here as well as in the other PR to `TRUSTSTORE`. Please let me know if it works for you and if it is more consistent.

Reply:
Yeah Thanks Jakub.